transactions/reset-epoch/2023/oct-25/arguments.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "type":"UInt64",
+    "value":"98"
+  },
+  {
+    "type":"String",
+    "value":"f28339976b22b61917e69fb5eca9eeb8"
+  },
+  {
+    "type":"UInt64",
+    "value":"9000000"
```

Comment:
Please double-check @joshuahannan @Kay-Zee before submitting the transaction.

Reply:
Lets do 8,900,000